rgw/v2/tests/s3_swift/test_sts_using_boto.py

```diff
@@ -117,6 +117,15 @@ def test_exec(config, ssh_con):
         log.info("put_policy_response")
         log.info(put_policy_response)
 
+        log.info(
```

Comment:
please update the config file in  section <input_yaml>

Reply:
added